chemprop/__init__.py

```diff
@@ -1,22 +1,13 @@
-# import chemprop.data
-# import chemprop.featurizers
-# import chemprop.models
-# import chemprop.train
-# import chemprop.uncertainty
-# # import chemprop.web
+import chemprop.cli
+import chemprop.data
+import chemprop.featurizers
+import chemprop.models
+import chemprop.nn
+import chemprop.utils
```

Comment:
same goes for the rest of the file:
```suggestion
from . import data, featurizers, models, nn, utils
```